projects/storefrontlib/shared/components/ng-select-a11y/ng-select-a11y.directive.ts

```diff
@@ -44,4 +62,24 @@ export class NgSelectA11yDirective implements AfterViewInit {
       this.renderer.setAttribute(divCombobox, 'aria-controls', ariaControls);
     }
   }
+
+  appendAriaLabelToOptions(
+    _changes: MutationRecord[],
+    observerInstance: MutationObserver
+  ) {
+    const options =
```

Comment:
We can use `HTMLOptionElement[]` to improve type safety and get rid of `any` below.